examples/nlp/machine_translation/nmt_transformer_infer.py

```diff
@@ -118,10 +118,24 @@ def main():
         help="Stop generating if target sequence length exceeds source length by this number.",
     )
     parser.add_argument(
-        "--target_lang", type=str, default=None, help="Target language identifier ex: en,de,fr,es etc."
+        "--target_lang",
+        type=str,
+        default=None,
+        help="Target language identifier ex: en,de,fr,es etc. If both `--target_lang` and `--source_lang` are "
+        "no set, then target language processing will be done the same way as during model training. If "
```

Comment:
Thanks for this detailed help message! Minor typo no set -> not set.

Reply:
@MaximumEntropy , I fixed typo